python/cudf/cudf/utils/dtypes.py

```diff
@@ -111,7 +122,7 @@ def cudf_dtype_from_pydata_dtype(dtype):
         )
```

Comment:
We could clean this up given we no longer support Pandas 0.23.x

Reply:
Done